cluster/addons/rbac/node-proxy-role.yaml

```diff
@@ -10,15 +10,9 @@ rules:
   - ""
   resources:
   - nodes/proxy
-  verbs:
-  - create
-  - get
-- apiGroups:
-  - ""
-  resources:
   - nodes/log
   - nodes/stats
   - nodes/metrics
   - nodes/spec
```

Comment:
Node-admin sounds to me like an ops role that would be provisioning/updating node objects...